src/main/java/in/ac/bits/protocolanalyzer/mvc/controller/TestController.java

```diff
@@ -38,24 +40,22 @@
 	@Qualifier("Experiment")
 	private Experiment exp1;
 
-//	@RequestMapping(method = RequestMethod.GET)
-	public void runSequentialExp()throws Exception {
-		String pcapPath = "/home/vikram/Darshini/Test_pcap_files/sample_capture_test1.pcap";
-		String protocolGraphStr = "graph start {\n\tethernet;\n}\ngraph ethernet {\n\tswitch(ethertype) {\n\t\tcase 0800:			 ipv4;\n\t}\n}\ngraph ipv4 {\n\tswitch(protocol) {\n\t\tcase 06: tcp;\n\t}\n}\ngraph tcp {\n}\ngraph end {\n}";
-
+	@RequestMapping(value = "/experiment", method = RequestMethod.GET)
```

Comment:
The [darshini-test.sh](https://github.com/prasadtalasila/BITS-Darshini/blob/dev/scripts/measurements/darshini-test.sh#L32) does not use "/test/experiment" path in the URL. We can safely remove the runSequentialExp() method.